docs/api/covidcast_changelog.md

```diff
@@ -70,26 +70,26 @@ NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now split in pro
 
 #### 7 October 2020
 
-The following changes were made to all `jhu-csse` signals related to geocoding
+The following changes were made to all `jhu-csse` signals related to geocoding:
 - NY Boroughs county FIPS (36005, 36047, 36061, 36081, 36085) are now [differentiated by JHU](https://github.com/CSSEGISandData/COVID-19/issues/3084),
 - “Unassigned” and “Out of State” counts are assigned to a megaFIPS at the county level (XX000 where XX is the state FIPS code) and will now be incorporated into the state totals
-- The split of the Kansas City, Missouri into its four subcounties is now done with a more precise population proportion,
-- The split of the Dukes and Nantucket, Massachusetts FIPS codes into the two subcounties is now done with a more precise population proportion,
-- Some counts are reported in the Utah territories JHU UIDs 84070015-84070020. These are now properly aggregated into the state megaFIPS 49000, which will be aggregated in state level information.
+- The split of the Kansas City, Missouri into its four sub-counties is now done with a more precise population proportion,
+- The split of the Dukes and Nantucket, Massachusetts FIPS codes into the two sub-counties is now done with a more precise population proportion,
+- Some counts are reported in the Utah territories JHU UIDs 84070015-84070020. These are now aggregated properly into the state megaFIPS 49000, which will be aggregated in state-level information.
 
 ### `indicator-combination`
 
 #### 3 June 2020
 
-Standard errors are now included in the `nmf_day_doc_fbc_fbs_ght` signal for all geo levels and dates, representing the estimated uncertainty in this signal. This uncertainty comes because the signal is a combination of the other signals which are based on survey estimates or other estimates with margins of error.
+Standard errors are now included in the `nmf_day_doc_fbc_fbs_ght` signal for all geo levels and dates, representing the estimated uncertainty in this signal. This uncertainty comes because the signal is a combination of the other signals based on survey estimates or other estimates with margins of error.
```

Comment:
```suggestion
Standard errors are now included in the `nmf_day_doc_fbc_fbs_ght` signal for all geo levels and dates, representing the estimated uncertainty in this signal. This uncertainty comes because the signal is a combination of other signals based on survey estimates or other estimates with margins of error.
```